src/cc2olx/olx.py

```diff
@@ -110,11 +110,22 @@ def _get_content(self, element_data):
         return content_type, details
 
     def _process_static_links(self, html):
-        srcs = re.findall(r'src\s*=\s*"(.+?)"', html)
-        for src in srcs:
-            if 'IMS-CC-FILEBASE' in src:
-                new_src = urllib.parse.unquote(src).replace("$IMS-CC-FILEBASE$", "/static")
-                html = html.replace(src, new_src)
+        """
+        This function helps to convert the IMSCC FILEBASE path
+        to a normal static path.
+
+        Args:
+            html ([str]): HTML content of the file.
+
+        Returns:
+            [str]: Corrected HTML content of the file.
+        """
+        # Here the attributes that are tagetted are href and src
+        srcs = re.findall(r'(src|href)\s*=\s*"(.+?)"', html)
+        for tag, link in srcs:
```

Comment:
It looks like `tag` is an unused variable here. Should we keep it around?

Reply:
it adds to the readability Josh so I think we should, I could make it `_` but that won't convey much.

Reply:
@kaizoku let me know what do you think about this? :)

Reply:
Awesome :)